Detectors/EMCAL/workflow/src/RawToCellConverterSpec.cxx

```diff
@@ -591,6 +602,16 @@ void RawToCellConverterSpec::run(framework::ProcessingContext& ctx)
   sendData(ctx, mOutputCells, mOutputTriggerRecords, mOutputDecoderErrors);
 }
 
+void RawToCellConverterSpec::finaliseCCDB(o2::framework::ConcreteDataMatcher& matcher, void* obj)
+{
+  // check if calib params need to be updated
+  if (matcher == framework::ConcreteDataMatcher("EMC", "EMCALRECOPARAM", 0)) {
+    LOG(info) << "RecoParams updated";
+    LOG(info) << o2::emcal::RecoParam::Instance();
```

Comment:
Why don't you use generic method `o2::emcal::RecoParam::Instance().printKeyValues(true, true);` ?
The 2nd true will ensure that the output goes to the LOG(info) instead of the stdout. The advantage is that also the provenance of the data member will be reported (e.g. in case you override some parameter from the command line).

Also, with your streaming operator I think only 1st line (data member) will be preceded by the [INFO], did you check it?

Reply:
I was not aware of this, I will try this and modify the PR accordingly.

Reply:
Code is changed to use `o2::emcal::RecoParam::Instance().printKeyValues(true, true);`, thanks a lot for the hint!